init.sql

```diff
@@ -0,0 +1,2 @@
+CREATE DATABASE IF NOT EXISTS shoppingking;
+USE shoppingking;
```

Comment:
nit: Please add a new line at the end of the file.
<img width="92" alt="image" src="https://github.com/f-lab-edu/shopping-King/assets/90605505/d701862a-5ed6-4c92-be34-a6ce2445b99f">